guide/src/commands/relaying/clear.md

```diff
@@ -0,0 +1,496 @@
+2022-02-24T14:17:33.215803Z  INFO ThreadId(15) send_tx{id=ibc0}: refresh: retrieved account sequence=62 number=1
+2022-02-24T14:17:33.245229Z  INFO ThreadId(15) wait_for_block_commits: waiting for commit of tx hashes(s) 62C69B1C46AF45182D5D99C6CB5EB301F8A402726772BA4EE067B18C68F2A4D6 id=ibc0
+2022-02-24T14:17:37.465489Z  INFO ThreadId(01) PacketAckCmd{src_chain=ibc1 src_port=transfer src_channel=channel-12 dst_chain=ibc0}:relay{odata=L4fnSXkxL_ ->Destination @0-86215; len=3}: [Sync->ibc0] result events:
+    UpdateClientEv(h: 0-86221, cs_h: 07-tendermint-3(0-86216))
+    AcknowledgePacketEv(h:0-86221, seq:14, path:channel-13/transfer->channel-12/transfer, toh:0-87203, tos:Timestamp(NoTimestamp)))
+    AcknowledgePacketEv(h:0-86221, seq:15, path:channel-13/transfer->channel-12/transfer, toh:0-87203, tos:Timestamp(NoTimestamp)))
+    AcknowledgePacketEv(h:0-86221, seq:16, path:channel-13/transfer->channel-12/transfer, toh:0-87203, tos:Timestamp(NoTimestamp)))
+
+
+2022-02-24T14:17:37.465802Z  INFO ThreadId(01) PacketAckCmd{src_chain=ibc1 src_port=transfer src_channel=channel-12 dst_chain=ibc0}:relay{odata=L4fnSXkxL_ ->Destination @0-86215; len=3}: success
+Success: [
+    UpdateClient(
+        UpdateClient {
+            common: Attributes {
+                height: Height {
+                    revision: 0,
+                    height: 86215,
+                },
+                client_id: ClientId(
+                    "07-tendermint-3",
+                ),
+                client_type: Tendermint,
+                consensus_height: Height {
+                    revision: 0,
+                    height: 86219,
+                },
+            },
+            header: Some(
+                Tendermint(
+                     Header {...},
+                ),
+            ),
+        },
+    ),
+    WriteAcknowledgement(
+        WriteAcknowledgement {
+            height: Height {
+                revision: 0,
+                height: 86215,
+            },
+            packet: Packet {
+                sequence: Sequence(
+                    14,
+                ),
+                source_port: PortId(
+                    "transfer",
+                ),
+                source_channel: ChannelId(
+                    "channel-13",
+                ),
+                destination_port: PortId(
+                    "transfer",
+                ),
+                destination_channel: ChannelId(
+                    "channel-12",
+                ),
+                data: [ ... ],
+                timeout_height: Height {
+                    revision: 0,
+                    height: 87203,
+                },
+                timeout_timestamp: Timestamp {
+                    time: None,
+                },
+            },
+            ack: [
```

Comment:
Can we skip this data like we did with `data: [ ... ],`.

Reply:
bd2706ac